src/main/java/org/apache/datasketches/Util.java

```diff
@@ -421,41 +421,38 @@ public static boolean isPowerOf2(final int v) {
    * Checks the given parameter to make sure it is positive, an integer-power of 2 and greater than
    * zero.
    *
-   * @param v The input argument.
+   * @param powerOf2 The input argument must be a power of 2 and greater than zero.
    * @param argName Used in the thrown exception.
    */
-  public static void checkIfPowerOf2(final int v, final String argName) {
-    if (v > 0 && (v & v - 1) == 0) {
-      return;
-    }
+  public static void checkIfPowerOf2(final int powerOf2, final String argName) {
+    if (powerOf2 > 0 && (powerOf2 & powerOf2 - 1) == 0) { return; }
     throw new SketchesArgumentException("The value of the parameter \"" + argName
-        + "\" must be a positive integer-power of 2" + " and greater than 0: " + v);
+        + "\" must be a positive integer-power of 2" + " and greater than 0: " + powerOf2);
   }
 
   /**
    * Checks the given value if it is a power of 2. If not, it throws an exception.
    * Otherwise, returns the log-base2 of the given value.
-   * @param value must be a power of 2 and greater than zero.
+   * @param powerOf2 must be a power of 2 and greater than zero.
    * @param argName the argument name used in the exception if thrown.
    * @return the log-base2 of the given value
    */
-  public static int toLog2(final int value, final String argName) {
-    checkIfPowerOf2(value, argName);
-    return Integer.numberOfTrailingZeros(value);
+  public static int exactLog2(final int powerOf2, final String argName) {
```

Comment:
Should this and the long version have a more consistent naming scheme? e.g. exactLog2OfInt() for this one?

Reply:
I'm following a java convention here that you will find in java internals.  The simpler form is for the method that is used most often.  I considered changing the other method to just exactLog2(long) since they don't collide. But it is used much less often so I figured why not just leave it alone.  If you insist on consistent naming between these two I will change the other one to exactLog2(long).

Reply:
Given the reasoning, I think I support your decision to go with this approach. No change needed.